app/views/content/train-to-be-a-teacher/if-you-dont-have-a-degree.md

```diff
@@ -15,6 +15,16 @@ promo_content:
 navigation: 20.30
 navigation_title: If you do not have a degree
 navigation_description: You need a degree to get qualified teacher status (QTS). If you're not already studying for one, find out more about undergraduate degree courses.
+expander:
+  international:
+    title: undergraduate degree courses
+    text: |-
+      <p>
+      If you're a non-UK citizen who wants to study in England, you can visit the <a href="https://www.ukcisa.org.uk/">UK Council for International Student Affairs</a> for information about studying at an English university.
+      </p>      
+  teacher-degree-apprenticeships:
```

Comment:
@emmaachesongray This needs removing from the front matter as well - sorry, missed this yesterday!